HopsanGenerator/templates/fmu_hopsan.cpp

```diff
@@ -25,9 +25,12 @@
 #include "fmu_hopsan.h"
 #include "HopsanCore.h"
 #include "HopsanEssentials.h"
+#include "HopsanTypes.h"
 #include <string>
 #include "model.hpp"
 #include <cassert>
+#include <iostream>
+#include <sstream>
```

Comment:
I cant compile (Kubuntu 19.10), #include <map> is missing
IS iostream really needed ? (and sstream should not be needed if to_hstring() can be used (See other comment)

Reply:
I fixed this as mentioned above.
Just for curiousity, why does this not work in Kubuntu? Is it a different compiler version?

Reply:
Yes likely due to different GCC version